### DISCUSSION TITLE
디자인 시스템에 대해서 | 정현수 기술 블로그

### DISCUSSION THREAD

**Discussion by giscus** [junghyeonsu/junghyeonsu.dev.comment#23] [Announcements]
# 디자인 시스템에 대해서 | 정현수 기술 블로그

디자인 시스템에 대한 확고한 가치관을 가지고 싶다.

https://junghyeonsu-dev.vercel.app/posts/About-design-system

**Comment by ds-package** [comment]
👍👍👍👍👍👍👍👍👍

**Reply by yeeeeeeeni** [reply]
좋은 인사이트 얻고갑니다~

**Comment by joseph-106** [comment]
디자인 시스템이라는 개념이 궁금했었는데, 재밌게 읽고 갑니다 ~~
